DEVELOPMENT.md

```diff
@@ -1,6 +1,7 @@
 ## Building JVM bindings
 
 * Set JAVA_HOME to location of JDK, at least version 11
+* If you have **Apple silicon CPU (M1)**, better to use **X86** JDK architecture to compile JS target.
```

Comment:
Doesn't it work otherwise? What happens otherwise?

Reply:
For me, on Arm64 JDK, sometimes I have error with yarn or webpack. (Like failed to install yarn)
I am also trying to use `kotlin.js.webpack.major.version=4`

With change JDK to X86 JDK, all works fine.
And for now, also Arm64 JDK works fine. Maybe it's a cache problem. I decided to mention JDK change as one of the possible solutions.

Reply:
I see. Thanks!

Reply:
I changed a little bit this sentence to better understanding.